pytest/lib/cluster.py

```diff
@@ -398,24 +398,25 @@ def start(self, *, boot_node: BootNode = None, skip_starting_proxy=False):
         except:
             logger.error(
                 '=== failed to start node, rpc does not ready in 10 seconds')
-            self.stdout.close()
-            self.stderr.close()
             if os.environ.get('BUILDKITE'):
```

Comment:
please comment what is the special thing that we need to do for 'BUILDKITE'

Reply:
With the other PR merged this PR no longer touches this piece of code so documenting this here feels out of place for this PR.